src/api/config/feature.yml

```diff
@@ -0,0 +1,2 @@
+features:
+  image_templates: true
```

Comment:
The only thing missing is to handle properly the YAML file in the spec, that's why I added the DO NOT MERGE label